File: DSA/revise.cpp
#include <iostream>
using namespace std;

int main()
{
    int num, dabba = 0, inhand = 1;
    cout << "Please enter a number: ";
    cin >> num;

    while (inhand <= num)
    {
        dabba = dabba + inhand;
        inhand += 1;
    }

    cout << "The sum from 1 to " << num << " = " << dabba << endl;

    return 0;
}

File: DSA/revise.java
import java.util.Scanner;



public class WhileNest {

    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);  
        System.out.print("Please enter a number: "); 
        int num = sc.nextInt(); 

        int dubba = 0;
        int hand = 1;
        while(hand <= num){
            dubba = dubba + hand;
            hand += 1;
            }

            System.out.println("The sum from 1 to " + num + " = " + dubba);
            
        }
        
    }




